Geometry/HcalCommonData/python/hcalDDConstants_cff.py

```diff
@@ -2,7 +2,6 @@
 
 from Geometry.HcalCommonData.hcalParameters_cff             import *
 from Geometry.HcalCommonData.hcalSimulationParameters_cff   import *
-from Geometry.HcalCommonData.hcalDDDSimConstants_cfi        import *
 from Geometry.HcalCommonData.hcalSimulationConstants_cfi    import *
-from Geometry.HcalCommonData.hcalDDDRecConstants_cfi        import *
 from Geometry.HcalCommonData.caloSimulationParameters_cff   import *
+from Geometry.HcalCommonData.hcalhcalDBConstants_cff        import *
```

Comment:
Does `hcalhcalDBConstants_cff` exist? The `hcalhcal` part seems odd.

Reply:
Corrected